environment.yml

```diff
@@ -17,7 +17,7 @@ dependencies:
   - openff-nagl-models>=0.1.2
   - openff-toolkit-base >=0.16.2
   - openff-units==0.3.1  # https://github.com/OpenFreeEnergy/openfe/pull/1374
-  - openmm ~=8.3.0
+  - openmm ~=8.2.0  # omit 8.3.0 and 8.3.1 due to https://github.com/openmm/openmm/pull/5069, unpin once we've qualified 8.3.2
```

Comment:
I prefer this over the `!=8.1.0,>=8.0.0,<8.3.0 ` use on the feedstock ([here](https://github.com/atravitz/openfe-feedstock/blob/5e513788f3ee3e6ac57939a2337a0f9fb578426a/recipe/meta.yaml#L43-L44))
because we want to know that things are building and testing with the latest version of openmm that we can.